packages/project/src/db/props.ts

```diff
@@ -49,23 +50,23 @@ export const loadByTreeId = async (treeId: Tree["id"]) => {
     const dpProps = UserDbProps.parse(props);
 
     for (const dbProp of dpProps) {
-      if (dbProp.assetId == null) {
-        userProps.push(dbProp);
+      if ("assetId" in dbProp) {
+        const { assetId, ...userProp } = dbProp;
+        const asset = assetsMap.get(assetId);
+
+        if (asset) {
+          userProps.push({
+            ...userProp,
+            asset,
+          });
+        } else {
```

Comment:
try to avoid using `else`, they always increase indentation and usually making it harder to read for someone who got used to early returns style.

In this case we can just use `continue;` before